### PR TITLE
Add ≤ ≥ ≠ unicode ops

### PR DESCRIPTION
**Pull request by AlexDaniel**
🙌

Tiny IRC discussion:
https://irclog.perlgeek.de/perl6/2017-03-01#i_14183585

⩵ (and especially ⩶) were left out intentionally. For now, let's focus
on operators that were actually requested many times.

Spectest clean.

### CODE REVIEW

**Comment by lizmat** [comment]
This makes these unicode operators inherently slower than their Texas counterparts.  Why not just create aliases?

  my &infix:<≠> := &infix:<!=>

??

**Comment by jnthn** [comment]
Because the alias is made a startup time if you use `:=`. Thus `&infix:<≠>` is considered a variable rather than a static symbol, and so optimizers must assume it can change. As a result, spesh will refuse to inline it, or even emit optimized calling code for it. You might get it to inline it if you alias it using `my constant ...`. But the way it is in the original patch is the only one I'm certain it's going to be able to inline.

**Comment by jnthn** [comment]
I did some measurements:

```
jnthn@lviv:~/edument/rmtly/try-it-out$ time perl6-m -e 'sub a($x, $y) { }; for ^1000000 { a(1,2) }'
real	0m0.815s
user	0m0.780s
sys	0m0.028s
jnthn@lviv:~/edument/rmtly/try-it-out$ time perl6-m -e 'sub a($x, $y) { }; my constant &b := &a; for ^1000000 { b(1,2) }'

real	0m0.889s
user	0m0.844s
sys	0m0.024s
jnthn@lviv:~/edument/rmtly/try-it-out$ time perl6-m -e 'sub a($x, $y) { }; my &b := &a; for ^1000000 { b(1,2) }'

real	0m1.055s
user	0m1.040s
sys	0m0.012s
jnthn@lviv:~/edument/rmtly/try-it-out$ time perl6-m -e 'sub a($x, $y) { }; sub b($x, $y) { a($x, $y) }; for ^1000000 { b(1,2) }'

real	0m1.278s
user	0m1.228s
sys	0m0.028s
```

The improvement of `constant` over `my` is pretty notable, making it the best way to go. At present, it also seems that the arg passing-on overhead is higher than the cost of the missed inline, so the originally suggested change would in the immediate term be an improvement. However, that is liable to improve, while the variable case is not.

tl;dr alias using constant.

**Comment by AlexDaniel** [comment]
I looked at how other things are implemented, and this is what I found:
* ``×`` and ``÷`` are implemented similarly to this PR ([src](https://github.com/rakudo/rakudo/blob/9cec31afee774e73dc49753e188c712bb450c1d4/src/core/Numeric.pm#L199)).
* All set ops are implemented like this as well ([src](https://github.com/rakudo/rakudo/blob/9cec31afee774e73dc49753e188c712bb450c1d4/src/core/set_operators.pm#L60))
* That's also how ``≅`` is implemented ([src](https://github.com/rakudo/rakudo/blob/9cec31afee774e73dc49753e188c712bb450c1d4/src/core/Numeric.pm#L285))
* ``−`` too ([src](https://github.com/rakudo/rakudo/blob/9cec31afee774e73dc49753e188c712bb450c1d4/src/core/Mu.pm#L965))

The only alternative op that is done differently is ``o`` ([src](https://github.com/rakudo/rakudo/blob/9cec31afee774e73dc49753e188c712bb450c1d4/src/core/operators.pm#L718)):
```perl6
my &infix:<o> := &infix:<∘>;
```

@lizmat do you think I should change it for this PR, or perhaps all alternative ops should be changed at once later?

**Comment by lizmat** [comment]
I would like to see that in a separate PR.  Possibly with timings  :-)

File: src/Perl6/Grammar.nqp [hunk 1]
```diff
@@ -4343,8 +4343,11 @@ grammar Perl6::Grammar is HLL::Grammar does STD {
     token infix:sym«≅»    { <sym>  <O(|%chaining)> }
     token infix:sym«==»   { <sym>  <O(|%chaining)> }
     token infix:sym«!=»   { <sym> <?before \s|']'> <O(|%chaining)> }
+    token infix:sym«≠»    { <sym> <?before \s|']'> <O(|%chaining)> }
```
Comment (AlexDaniel) [inline src/Perl6/Grammar.nqp]: True. I was thinking about that too. Fixed now.

**Comment by AlexDaniel** [comment]
@lizmat yea, I think I can do that.

**Comment by zoffixznet** [comment]
Thanks.
